app/models/outgoing_email.rb

```diff
@@ -38,6 +38,7 @@ class OutgoingEmail < ApplicationRecord
   validates_presence_of :body
   validates_presence_of :subject
   validates :mailgun_status, inclusion: { in: ALL_KNOWN_MAILGUN_STATUSES }
+  attr_accessor :locale, :tax_return
```

Comment:
I'm not sure I understand why this is needed, and would love to. Not a blocker.

Reply:
i feel like i removed this from the other PR but had already branched off of it so it got added back. will remove.